### PR TITLE
refactor: download icons

### PR DESCRIPTION
**Pull request by edikdeisling**
### Type of change

- [ ] Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] Bug fix (a non-breaking change that fixes an issue)
- [ ] Enhancement (improving an existing functionality like performance)
- [ ] New feature (a non-breaking change that adds functionality)
- [x] Refactoring (old API is revised and supported)
- [ ] Chore (updates to the build process or auxiliary tools and libraries)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have linked an **issue** or **discussion**;
- [ ] I have updated the **documentation** accordingly;
- [ ] I have added or updated **tests** to cover my changes;
- [ ] I have performed a **self-review** of my code;
- [ ] My code follows the style **guidelines** of this project;
- [ ] My changes generate no new **warnings**.

### CODE REVIEW

File: packages/icons/.env [hunk 1]
```diff
@@ -1,6 +1,10 @@
-FIGMA_FETCHER_FILE_KEY="KEY_EXAMPLE"
-FIGMA_FETCHER_FIGMA_TOKEN="figd_EXAMPLE"
-FIGMA_FETCHER_PAGE_NAMES="Actions, Arrows"
+# figma file key, should be set in .env.local
+FIGMA_FETCHER_FILE_KEY=SeWLzXxzzYfvfjzzbTYUl1
```
Comment (leonid) [inline packages/icons/.env:2]: This looks like real key, please use fake one

Comment (Jerd0) [reply packages/icons/.env:2]: this key - just figma id, why its forbidden?

Comment (edikdeisling) [reply packages/icons/.env:2]: @leonid we have this key inside https://github.com/acronis/ui-component-library/blob/main/packages/documentation/guide/onboarding.md#useful-links

I thought it's not private

Comment (leonid) [reply packages/icons/.env:2]: > this key - just figma id, why its forbidden?

We have security check that will throw an error on possible private keys. But yes in that case we already have in docs. So you can keep it and we will add to ignore file if needed later.

**Review by leonid — APPROVED** [review]

**Pull request merged**